tests/console/yast2_proxy.pm

```diff
@@ -104,6 +112,17 @@ sub run {
     # move to http ports
     select_sub_menu 'start_up', 'http_ports';
 
+    # add a forwarding port
```

Comment:
Does it also work on SLE 12 SP4? And I guess we need MR for SLE needles.

Reply:
I'll get back to you on this one. It *should*

Reply:
Yes, works just fine with SLE 12 SP4 tests.

http://tortuga.suse.de/tests/169

Reply:
Great, I'll merge then. Thanks!